plugins/textpattern.md

```diff
@@ -31,7 +31,7 @@ This option allows configuring the text patterns that get matched by the `textpa
 
 There are three types of patterns: `inline`, `block`, and `replacement` patterns. Inline patterns have a `start` and an `end` text pattern whereas the block and replacement patterns only have a `start`. A user can specify the formats to be applied to the selection, commands to be executed, or text to be replaced.
 
-When using list commands make sure that the [lists plugin]({{ site.baseurl }}/plugins/lists) is registered for normalized behavior across browsers.
+**When using list commands make sure that the [lists plugin]({{ site.baseurl }}/plugins/lists) is registered for normalized behavior across browsers.**
```

Comment:
Why was this made bold? This seems incorrect... I assume it was done to highlight this, if that was why it should be an admonition/note instead of being bold. eg:
```
> **Note:** Some text
```

Reply:
You are right, it was put there to highlight it. As i'm new to this maybe a less intimidated way to put that would be... "We shouldn't use it like that, etc etc...: :-) Thanks

Reply:
Done